src/sgame/sg_cmds.cpp

```diff
@@ -4442,6 +4451,58 @@ void Cmd_PrivateMessage_f( gentity_t *ent )
 	}
 }
 
+
+/*
+=================
+Cmd_ReplyPrivateMessage_f
+
+Reply by private message to the last private message sender
+=================
+*/
+
+void Cmd_ReplyPrivateMessage_f(gentity_t *ent)
+{
+	char *msg;
+	int target;
+	bool teamonly = false;
+
+	if (!g_privateMessages.Get() && ent)
+	{
+		ADMP( "\"" N_("Sorry, but private messages have been disabled") "\"" );
+		return;
+	}
+
+	if (trap_Argc() < 2)
+	{
+	ADMP( QQ( N_("usage: /r [message]") ) );
```

Comment:
Maybe it would be a good idea to print who we are replying to here?

Reply:
that's to say ?

Reply:
The suggestion being made here is to show who lastPrivateMessageSender is. In my opinion you don't have to do it

Reply:
oh. I don't think that it is a good idea. Well, this is the usage. It just say "how to use the command"